docs/api-guide.md

```diff
@@ -708,14 +704,14 @@ public class Sample {
         TimeUnit.MILLISECONDS.sleep(100);
       }
 
-      // Start a transaction
+      // Begin a transaction
       DistributedTransaction tx;
       try {
-        tx = manager.start();
+        tx = transactionManager.begin();
       } catch (TransactionException e) {
-        // If starting a transaction fails, it indicates some failure happens during a transaction,
-        // so you should cancel the transaction or retry the transaction after the failure/error is
-        // fixed
+        // If starting a transaction fails, it indicates some failure happens during the
```

Comment:
```suggestion
        // If beginning a transaction fails, it indicates some failure happens during the
```
To stay consistent with the change from `start()` to `begin()` in the code right above.